### ISSUE TITLE
All HRTF data files report being for 44100 Hz sample rate

### ISSUE THREAD

**Issue by forrcaho** [csound/csound#1379]
I've tried hrtfstatic, hrtfmove, and hrtfmove2 in all combinations of specifying sr as 44100, 48000, or 96000 and specifying the data files from the samples dir for 44100, 48000, and 96000.

Whenever I specify a non-44100 sr, no matter which data files I specify, I get this warning message:

WARNING!!:
Orchestra SR not compatible with HRTF processing SR of: 44100

Conversely, if I specify sr=44100, I _don't_ see this message, no matter which data files are specified.

In other words, csound is reporting that all the data files are for a 44100 Hz.

I built a fresh csound from today's git sources to test this. I'm running on Debian linux "unstable".

I'm also very dubious about whether the effects of these opcodes are anything like their descriptions, but that is a separate issue.

**Comment by jpffitch** [comment]
I do not see that.  I regularly use 48000 for sr and use the hrtf files to 
match

**Comment by forrcaho** [comment]
Here is a complete sample CSD file:
```
<CsoundSynthesizer>
<CsInstruments>

sr = 48000
ksmps = 32
nchnls = 2
0dbfs = 1

instr 1
iFreq = p4
iAmpl = ampdbfs(p5)

kAz line p6, p3, p7
kElev line p8, p3, p9 

aSrc poscil3 iAmpl, iFreq
aLeft, aRight hrtfmove aSrc, kAz, kElev, "hrtf-48000-left.dat", "hrtf-48000-right.dat"
outs aLeft, aRight
endin

</CsInstruments>
<CsScore>
i1 0 4 200 -6 -60 60 -30 30
</CsScore>
</CsoundSynthesizer>
```
I have copied the hrtf data files from the samples dir of the csound github repo into my current working directory.
Here is the output I get from running the above csound code:
```
forrcaho@vortex:~/musicode/csound/hrtf$ csound hrtftest.csd 
0dBFS level = 32768.0
--Csound version 6.14 (double samples) 2020-06-11
[commit: none]
libsndfile-1.0.28
UnifiedCSD:  hrtftest.csd
STARTING FILE
Creating orchestra
closing tag
Creating score
rtaudio: ALSA module enabled
rtmidi: ALSA Raw MIDI module enabled
Elapsed time at end of orchestra compile: real: 0.002s, CPU: 0.002s
sorting score ...
	... done
Elapsed time at end of score sort: real: 0.002s, CPU: 0.002s
graphics suppressed, ascii substituted
0dBFS level = 1.0
orch now loaded
audio buffered in 256 sample-frame blocks
writing 1024-byte blks of shorts to test.wav (WAV)
SECTION 1:
new alloc for instr 1:


WARNING!!:
Orchestra SR not compatible with HRTF processing SR of: 44100

file hrtf-48000-left.dat (188416 bytes) loaded into memory
file hrtf-48000-right.dat (188416 bytes) loaded into memory
B  0.000 ..  4.000 T  4.000 TT  4.000 M:  0.24600  0.25722
Score finished in csoundPerform().
inactive allocs returned to freespace
end of score.		   overall amps:  0.24600  0.25722
	   overall samples out of range:        0        0
0 errors in performance
Elapsed time at end of performance: real: 0.038s, CPU: 0.038s
512 1024 sample blks of shorts written to test.wav (WAV)
```
This _does_ produce a valid output wav file, but it seems that the sound is static in the original position, I don't hear it move at all in my headphones. I thought I would try to figure out what the "SR not compatible" warning was about first, before looking into that issue.

**Comment by jpffitch** [comment]
First, may I apologise for the time it has taken to look at this?

I have fixed this; there was some misguided code to give a value to sr (last parameter) if it was omitted.  it now defaults to the orchestra's sr.  One could argue it should take the default from the data file but .....  well rhatwould be a little harder!

Change committed to github now

Your example has a problem I think is unrelated but the output generated inf values and a lot of out-of-range values.
